source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+**Per-app VPN**
+    In contrast, the more common approach is to use a per-app VPN. This provides a connection to the VPN when needed (on-demand). For example, when using a particular app.
+
+Regardless of the commonness of either option above, Mattermost provides support for both. Because Mattermost supports the AppConfig standard, both options are compatible with EMM providers.
+
+    **Note:** Will you be connecting via a corporate proxy server? If so, `review our FAQ <https://docs.mattermost.com/mobile/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`_ covering architecture, troubleshooting and best practices.
+
+----
+
+##########################################
+Choosing the Right Mobile Deployment Model
+##########################################
+
+At this point you should have read through the deployment primer. It provides a large amount of context for the principles and best practices that follow. This next section aims to help you choose one of two recommended deployment models.
+
+*****************************************
+Using the Public App Stores (Recommended)
+*****************************************
+
+**Desired Outcomes:**
+
+- Quick rollout of the Mattermost mobile application.
+- Allow users to install the application on their device.
+- Ensure a high level of security and controlled access.
+- Use existing, internal processes and tools.
+
+We recommend either of these options if you are…
+
+- Testing out the mobile applications
+- Deploying Team Edition servers using no push notifications, or push notifications from Mattermost’s `TPNS <https://docs.mattermost.com/overview/faq.html#tpns>`_ (Test Push Notification Service)
+- Deploying Enterprise Edition servers using push notifications from Mattermost’s `HPNS <https://docs.mattermost.com/mobile/mobile-hpns.html>`_ (Hosted Push Notification Service)
+
+The mobile applications provided by Mattermost work with our hosted version of the Mattermost Push Notification Service (MPNS). This represents the easier path. The Mattermost mobile applications can be deployed with or without an EMM provider. These options are explained in more detail below. 
+
+    **Note:** HPNS is compatible with EMM providers.
+
+**Option 1** - Public App Store Installation (Easiest)
+
+- Users download application via public app stores (e.g. iOS App Store or Google Play)
+- Users enter URL to your hosted Mattermost Server.
+
+**Advantages:** Very easy, mobile deployment can be done by each individual user 
+**Disadvantages:** Less control over user security, no app customization
+
+**Option 2** - Public App Store Installation with EMM Provider (Easy)
+
+- Users download application via public app stores (e.g. iOS App Store or Google Play)
```

Comment:
not true, user downloads the app from the EMM catalog. in the event users download the apps from google play or apple app store, the app can become managed by the EMM

Reply:
I’m totally open to being wrong, but our documentation seems to convey exactly this. “ Mattermost mobile apps can be configured in your EMM solution using AppConfig with the apps on the public app stores (Google Play and Apple App Store), or as an “in-house app” you compile yourself. https://docs.mattermost.com/mobile/mobile-appconfig.html#other-appconfig-settings

I can see where there may be some semantics around how the apps were obtained. However, (totally open to I misunderstood) when speaking with @johnthompson365 and others we wanted to put this forward as an option. Is this a case of it’s not possible? I think we’d want to be very clear in the docs as well if this is not the case.

Reply:
This section describes the option with the following use case:
1. Using the mobile apps distributed by Mattermost via Google Play and the Apple App Store.
2. Using the HPNS
3. Desire to use an EMM provider to further control security parameters via an EMM.

In your first item, if they make the apps available via the EMM catalog, they would have to rebuild them correct? Or is there a way to somehow by pass the Google and Apple stores and point towards the EMM catalog. That seems impossible, but maybe I missed that somehow?

Reply:
resolved in meeting